tornado/log.py

```diff
@@ -126,6 +135,10 @@ def __init__(self, color=True, fmt=DEFAULT_FORMAT,
             for levelno, code in colors.items():
                 self._colors[levelno] = unicode_type(curses.tparm(fg_color, code), "ascii")
             self._normal = unicode_type(curses.tigetstr("sgr0"), "ascii")
+        elif colorama is not None:
```

Comment:
This also needs to be conditioned on the `color` variable and `_stderr_supports_color()`. Make this 

```
    if color and _stderr_supports_color():
        if curses is not None:
           ...
        elif colorama is not None:
           ..
        else:
            raise RuntimeError("no supported color terminal library")
```